pinecone/config/openapi_config_factory.py

```diff
@@ -72,7 +70,8 @@ def _get_socket_options(
         """
         # Source: https://www.finbourne.com/blog/the-mysterious-hanging-client-tcp-keep-alives
 
-        socket_params = HTTPConnection.default_socket_options
+        # urllib3.connection.HTTPConnection.default_socket_options
```

Comment:
This config gets evaluated during intialization, and loading the entire urllib3 package just to get access to a reference to these socket constants was adding a bit penalty to our load time.